pom.xml

```diff
@@ -394,18 +369,18 @@
               </licenseFamily>
             </licenseFamilies>
             <excludes>
+              <exclude>NOTICE</exclude>
               <exclude>**/.*</exclude>
-              <exclude>**/*.txt</exclude>
-              <exclude>**/*.sh</exclude>
-              <exclude>**/*.log</exclude>
+              <exclude>**/*.json</exclude>
+              <exclude>**/*.sqltemplate</exclude>
+              <exclude>**/compose_env</exclude>
+              <exclude>**/*NOTICE*</exclude>
+              <exclude>**/*LICENSE*</exclude>
               <exclude>**/dependency-reduced-pom.xml</exclude>
-              <exclude>**/test/resources/*.avsc</exclude>
               <exclude>**/test/resources/*.data</exclude>
-              <exclude>**/test/resources/*.schema</exclude>
-              <exclude>**/test/resources/*.csv</exclude>
-              <exclude>**/main/avro/*.avsc</exclude>
-              <exclude>**/target/*</exclude>
+              <exclude>**/target/**</exclude>
               <exclude>**/style/*</exclude>
+              <excludes>**/generated-sources/**</excludes>
```

Comment:
style are css? those also should have license headers.

Reply:
@lresende : this is for checkstyle and code-formatting rules. We can remove this as the underlying xml files are not auto-generated

Reply:
Fixed.